App.tsx

```diff
@@ -165,13 +166,13 @@ const BaseApp: React.FunctionComponent<{
                   }
                 },
               })}>
-              <TabNavigator.Screen name="Home" initialParams={{center_id: avalancheCenterId, date: date}}>
+              <TabNavigator.Screen name="Home" initialParams={{center_id: avalancheCenterId, dateString}}>
```

Comment:
If you are setting an object key with a variable of the same name, you’re allowed to just type the name once, instead of `dateString: dateString`. 

I guess this is ES6 syntax maybe? It’s not just typescript…

Reply:
https://eslint.org/docs/latest/rules/object-shorthand